packages/block-editor/src/components/block-heading-level-dropdown/stories/index.story.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import HeadingLevelDropdown from '../';
+
+export default {
+	title: 'BlockEditor/HeadingLevelDropdown',
+	component: HeadingLevelDropdown,
+	argTypes: {
+		value: {
+			control: { type: 'number', min: 0, max: 6, step: 1 },
+			description: 'The currently selected heading level.',
+		},
```

Comment:
Unsure if this is intentional or not, but I noticed changing the `value` control does not have an impact on the rendered component—i.e., you cannot change the currently selected dropdown item by modifying this control. 

Should this control be fixed, removed, or renamed to `initialValue` (to communicate its nature)?